### DISCUSSION TITLE
이거 풀게요 12월 네번째 주

### DISCUSSION THREAD

**Discussion by brewagebear** [brewagebear/ISOLVEDTHEPROBLEM#19] [금주의 문제]
**금주의 챌린지 문제**

1. [완주 못한 선수](https://programmers.co.kr/learn/courses/30/lessons/42576) - 레벨 1
2. [오픈 채팅방](https://programmers.co.kr/learn/courses/30/lessons/42888) - 레벨 2

**개인적으로 풀 문제(참가자 알아서 임의의 문제 설정)**

레벨 1 시간 나는 대로 다 풀기

**Comment by csj4032** [comment]
[멀쩡한 사각형](https://programmers.co.kr/learn/courses/30/lessons/62048)
어떤 삽질을 했는지 얘기를 해보겠습니다. :)

**Comment by pch8388** [comment]
개인적으로 푼 문제

[로또의 최고 순위와 최저 순위](https://programmers.co.kr/learn/courses/30/lessons/77484)

**Comment by tate-git** [comment]
[완주하지 못한 선수](https://programmers.co.kr/learn/courses/30/lessons/42576)
[오픈채팅방](https://programmers.co.kr/learn/courses/30/lessons/42888)

멀쩡한 사각형 읽고 고민중입니다~!
